drivers/luna/device.js

```diff
@@ -90,6 +93,7 @@ class Huawei extends Device {
                 const devRealKpiInverter = await lunaApi.getDevRealKpi(devListInverterId.id, devListInverterId.devTypeId, server);
                 // console.log(devRealKpiInverter);
                 await this.setCapabilityValue('meter_power.sun_power', devRealKpiInverter.mppt_power);
+                await this.setCapabilityValue('measure_power', devRealKpiInverter.mppt_power);
                 this.setStoreValue("sun_power", devRealKpiInverter.mppt_power);
```

Comment:
you are setting the same value here, it is the same as on line 95 `devRealKpiInverter.mppt_power`

Reply:
Yes it is the same value but for a different sensor so we can use as energy value. Don't want to change sun since some people already have insights on it.

Reply:
Will actually change to another value which is more correct.